packages/elements-react/src/context/form-state.ts

```diff
@@ -46,6 +47,12 @@ function parseStateFromFlow(flow: OryFlowContainer): FormState {
       ) {
         return { current: "method_active", method: flow.flow.active }
       } else if (isChoosingMethod(flow.flow.ui.nodes)) {
+        // Login has a special case where we only have one method. Here, we
```

Comment:
Does this also work for registration? This is executed on registration flows as well.

Reply:
Probably

Reply:
```suggestion
        // Login and registration have a special case where we only have one method. Here, we
```